dotnet/src/Connectors/Connectors.OpenAIV2/Extensions/ClientResultExceptionExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+/* 
+Phase 01:
+This class is introduced in exchange for the original RequestExceptionExtensions class of Azure.Core to the new ClientException from System.ClientModel,
+Preserved the logic as is.
+*/
+
+using System.ClientModel;
+using System.Net;
+
+namespace Microsoft.SemanticKernel.Connectors.OpenAI;
```

Comment:
Should this be OpenAI specific?

Reply:
For now, yes, but when this becomes a utility will remove the namespace.